depends/patches/qt/no-xlib.patch

```diff
@@ -53,7 +53,7 @@ index 7c62c2e2b3..c05c6c0a07 100644
      cursor = createNonStandardCursor(cshape);
  
 +#if QT_CONFIG(xcb_xlib) && QT_CONFIG(library)
-     // Create a glpyh cursor if everything else failed
+     // Create a glyph cursor if everything else failed
```

Comment:
You can't modify the actual contents of a patch like this, because it will no longer apply. See all the CIs that are now failing:
```bash
patching file qtbase/src/plugins/platforms/xcb/qxcbcursor.cpp
Hunk #5 FAILED at 595.
1 out of 6 hunks FAILED -- saving rejects to file qtbase/src/plugins/platforms/xcb/qxcbcursor.cpp.rej
funcs.mk:267: recipe for target '/tmp/cirrus-ci-build/depends/work/build/x86_64-w64-mingw32/qt/5.9.8-ed5ca6e91a4/.stamp_preprocessed' failed
make: *** [/tmp/cirrus-ci-build/depends/work/build/x86_64-w64-mingw32/qt/5.9.8-ed5ca6e91a4/.stamp_preprocessed] Error 1
make: Leaving directory '/tmp/cirrus-ci-build/depends'
```